client/client.go

```diff
@@ -429,16 +429,14 @@ func (c *client) loadKeyspaceMeta(keyspace string) error {
 }
 
 func (c *client) setup() error {
+	// Init the metrics.
+	initAndRegisterMetrics(c.option.metricsLabels)
+
 	// Init the client base.
 	if err := c.pdSvcDiscovery.Init(); err != nil {
 		return err
 	}
 
-	// Init the metrics.
-	if c.option.initMetrics {
```

Comment:
Why remove it?

Reply:
It shouldn't add c.option.initMetrics, because it will cause panic.

Reply:
Hi Zheming, regarding ​[client: add a option to control whether to initialize the metric when… · tikv/pd@b159dd2](https://github.com/tikv/pd/commit/b159dd2df37c8da4c001ec176b4df706a78ea2bb)
```
        // Init the metrics.
        if c.option.initMetrics {
                initAndRegisterMetrics(c.option.metricsLabels)
        }
```
We added "c.option.initMetrics" to call initAndRegisterMetrics conditionally, but Cmd Durations are always used without checking if it is initialized or not, will it cause null reference panic if the caller turns off c.option.initMetrics when creating a pd client?

Reply:
Zheming, confirmed "It will panic, I revert this PR first." and has a follow-up to fix it. https://github.com/tikv/pd/pull/6441

Reply:
I added back so that my pr is independent of Zheming's fix